requirements-doc.txt

```diff
@@ -3,6 +3,8 @@ sphinx
 matplotlib
 sphinx_rtd_theme
 sphinx-gallery
-allensdk>=2.11.0  # python 3.8 is not supported in allensdk<2.11
+h5py
+pynwb
+#allensdk>=2.11.0  # python 3.8 is not supported in allensdk<2.11
```

Comment:
why comment out allensdk? It is needed for the allensdk tutorial which is used in https://pynwb.readthedocs.io/en/stable/tutorials/domain/brain_observatory.html#sphx-glr-tutorials-domain-brain-observatory-py

Reply:
oh, yeah, I need to do this locally to get the docs to build. We can uncomment for the PR though